### ISSUE TITLE
PyTorchWrapper compile_kwargs are ignored

### ISSUE THREAD

**Issue by olineumann** [KIT-IAI/pyWATTS#33]
**Describe the bug**
When passing `compile_kwargs` dict to the PyTorchWrapper they are ignored. For example when choosing different metrics, optimizers, or loss functions.

**To Reproduce**
I tried to use a neural network for classification by using cross-entropy as loss and metric function. Switching the optimizer also did not work.

**Expected behavior**
`compile_kwargs` dict values are used as metric, loss, optimizer, ...

**Screenshots**
See code in pytorch_wrapper.py:
```python
        # check if gpu is available
        device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
        self.model.to(device)

        # loss_str = self.compile_kwargs['loss']
        # optimizer_str = self.compile_kwargs['optimizer']
        # metrics_str = self.compile_kwargs['metrics']

        batch_size_str = self.fit_kwargs['batch_size']
        epochs_str = self.fit_kwargs['epochs']

        x_np = xarray_to_numpy(x)
        y_np = xarray_to_numpy(y)

        dataset = TimeSeriesDataset(x_np, y_np)
        train_loader = DataLoader(dataset=dataset, batch_size=int(batch_size_str), shuffle=True)

        loss_fn = torch.nn.CrossEntropyLoss()

        learning_rate = 1e-4
        optimizer = torch.optim.Adam(self.model.parameters(), lr=learning_rate)
        scheduler = StepLR(optimizer, step_size=1)
```
